configure

```diff
@@ -733,6 +733,19 @@ else
   echo "Checking for strerror... No." | tee -a configure.log
 fi
 
+# check for getauxval() for architecture feature detection at run-time
+cat > $test.c <<EOF
+#include <sys/auxv.h>
+int main() { return getauxval(0); }
```

Comment:
This one is passing in 0 to `getauxval` but the cmake version is passing in other values..

Reply:
That's because here we only test that the function exists... Later we test if the feature macros are defined, as those are not always defined in `sys/auxv.h`.